lib/fathead/npm_cli/parse.py

```diff
@@ -0,0 +1,216 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+import requests
+import re
+from bs4 import BeautifulSoup
+
+
+START_URL = 'https://docs.npmjs.com/cli/install'
+BASE_URL = 'https://docs.npmjs.com'
+OUTPUL_FILE = open('output.txt', 'w+')
+RESULT_URLS = []
+
+
+# Print the content to the output file
+def print_article_line(title, code, content):
+
+    title = 'npm-' + title
+    code = code.replace('\n', '\\n').replace('\t', '  ')
+    code = code.replace('<', '&lt;').replace('>', '&gt;')
+    content = content.replace('\n', '\\n').replace('\t', '  ')
+    content = content.replace('<', '&lt;').replace('>', '&gt;')
+
+    abstract = '<section class="prog__container">' + content + \
+               '<code><pre>' + code + '</pre></code></section>'
```

Comment:
@rahulxxarora this is the wrong order: it should be `<pre><code>`, not `<code><pre>` 👍